configure.ac

```diff
@@ -802,6 +802,23 @@ dnl ========================================================================
 AC_CHECK_FUNCS(getopt, AC_DEFINE(HAVE_DECL_GETOPT,  1, [Have getopt function]))
 AC_CHECK_FUNCS(nanosleep, AC_DEFINE(HAVE_DECL_NANOSLEEP,  1, [Have nanosleep function]))
 
+AC_CACHE_CHECK(whether sscanf supports %m,
+               pf_cv_var_sscanf,
+               AC_RUN_IFELSE([AC_LANG_SOURCE([[
+#include <stdio.h>
+const char *s = "some-command-line-arg";
+int main(int argc, char **argv) {
+char *name = NULL;
+int n = sscanf(s, "%ms", &name);
+return n == 0 ? 1 : 0;
```

Comment:
sscanf() can return EOF (-1) ... even though that should never happen here, it would be safer to check n == 1 as the only success